src/Transports/MassTransit.ActiveMqTransport/ActiveMqTransport/Middleware/ActiveMqConsumerFilter.cs

```diff
@@ -47,8 +47,12 @@ async Task IFilter<SessionContext>.Send(SessionContext context, IPipe<SessionCon
 
             var supervisor = CreateConsumerSupervisor(context, actualConsumers);
 
+            await supervisor.Ready.ConfigureAwait(false);
```

Comment:
Why are you touching ActiveMQ?

Reply:
If you would split the commit so that two completely unrelated changes are easy to view in the history that would be great!